diff --git a/node_modules/bulma/sass/base/generic.sass b/node_modules/bulma/sass/base/generic.sass
index 31eedfe..eab8c4b 100644
--- a/node_modules/bulma/sass/base/generic.sass
+++ b/node_modules/bulma/sass/base/generic.sass
@@ -94,7 +94,6 @@ hr
   margin: $hr-margin
 
 img
-  height: auto
   max-width: 100%
 
 input[type="checkbox"],
diff --git a/node_modules/bulma/sass/base/minireset.sass b/node_modules/bulma/sass/base/minireset.sass
index c5657eb..4c3f137 100644
--- a/node_modules/bulma/sass/base/minireset.sass
+++ b/node_modules/bulma/sass/base/minireset.sass
@@ -3,9 +3,6 @@
 html,
 body,
 p,
-ol,
-ul,
-li,
 dl,
 dt,
 dd,
@@ -36,10 +33,6 @@ h6
   font-size: 100%
   font-weight: normal
 
-// List
-ul
-  list-style: none
-
 // Form
 button,
 input,
@@ -60,7 +53,6 @@ html
 // Media
 img,
 video
-  height: auto
   max-width: 100%
 
 // Iframe
